lib/assets.js

```diff
@@ -29,33 +29,75 @@ const getEmittedAssets = () => glob
  * Returns an array of assets (files + metadata).
  * @name getDistAssets
  * @param {boolean} includeContent
+ * @param {boolean} isStaticClient
  * @returns {Array} assets
  */
-const getDistAssets = (includeContent) => {
-  const metadataPath = path.join(dist, 'metadata.json');
+const getDistAssets = (includeContent, isStaticClient, version) => {
+  let metadataPath = path.join(dist, 'metadata.json');
+
+  if (isStaticClient) {
+    return glob.sync(path.join(dist, 'bundles', '*.js'))
```

Comment:
This looks really good to me @Blackbaud-TimPepper.  I think I have one small last change, I would go ahead and add a line (around line 12) like `const bundlesDist = path.join(dist, 'bundles')` and use that here and line 73.  That way **if** we ever change the folder, it's only in one spot.